docs/src/test/scala/docs/http/scaladsl/server/WebSocketExampleSpec.scala

```diff
@@ -122,14 +122,10 @@ class WebSocketExampleSpec extends AnyWordSpec with Matchers with CompileOnlySpe
     val defaultSettings = ServerSettings(system)
 
     val pingCounter = new AtomicInteger()
-    val customWebsocketSettings =
-      defaultSettings.websocketSettings
-        .withPeriodicKeepAliveData(() => ByteString(s"debug-${pingCounter.incrementAndGet()}"))
-
-    val customServerSettings =
-      defaultSettings.withWebsocketSettings(customWebsocketSettings)
 
-    Http().bindAndHandle(route, "127.0.0.1", settings = customServerSettings)
+    Http().newServerAt("127.0.0.1")
+      .adaptSettings(_.mapWebsocketSettings(_.withPeriodicKeepAliveData(() => ByteString(s"debug-${pingCounter.incrementAndGet()}"))))
```

Comment:
That's not super nice. We could introduce first-class lenses for those at some point. Or we could at least add an easy way to add a string based config override (which wouldn't help here).

Reply:
(agreed, though it's not much worse than before)